modules/apps/blogs/blogs-editor-configuration/src/main/java/com/liferay/blogs/editor/configuration/internal/BlogsContentEditorConfigContributor.java

```diff
@@ -82,6 +85,19 @@ public void populateConfigJSONObject(
 			namespace + name + "selectItem");
 
 		_populateTwitterButton(jsonObject);
+
+		PortletDisplay portletDisplay = themeDisplay.getPortletDisplay();
```

Comment:
I'm curious about this... why do we need to change it from the optionscontributor to here? 

They are similar ways to pass things, just that one becomes part of the `config` and the other part of additional options...

I don't see a problem with this, though, just curious.

Reply:
`config` is a CKEditor native JS object that we populate in a `ConfigContributor`. `editorOptions` is a Liferay-specific Java object that we populate in `OptionsContributor`. In a JSP we can [access both](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp#L43-L53), as they are forwarded through `data` attribute, but from a plugin we can only access `config`.

Reply:
Yeah, I know that... what I was wondering about is how it was being passed before in the plugin. Turns out, it wasn't, we were simply using it to instantiate the AUI component.